### PR TITLE
[Cosmos] Remove unused files

### PR DESCRIPTION
**Pull request by southpolesteve**
- Remove unused murmur hash and consistent hashing ring files
- Run local tests against proper emulator port

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/test/test_config.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class _test_config(object):
 
     #[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Cosmos DB Emulator Key")]
     masterKey = os.getenv('ACCOUNT_KEY', 'C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==')
-    host = os.getenv('ACCOUNT_HOST', 'https://localhost:443/')
+    host = os.getenv('ACCOUNT_HOST', 'https://localhost:8081/')
```
Comment (southpolesteve) [inline sdk/cosmos/azure-cosmos/test/test_config.py:42]: The internal emulator uses 443. This change runs the tests against the public emulator on 8081

**Comment by southpolesteve** [comment]
@annatisch What's the difference between the tests that ran when I opened the PR and the ones that ran when you triggered the pipeline via the bot?

**Comment by annatisch** [comment]
@southpolesteve - it should be live vs emulator.
However it looks like there's currently an issue without our engineering system (the OSName error). I will re-trigger once that's been resolved on our side :)

**Pull request merged**